src/EditorFeatures/CSharpTest/Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj

```diff
@@ -62,14 +62,6 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
-  <!-- These are None\content items above. Don't pass them to the compiler. -->
-  <ItemGroup>
-    <Compile Remove="Interactive\CommandArgumentsParsingTest.cs" />
-    <Compile Remove="ReorderParameters\ReorderParametersTests.Cascading.cs" />
-    <Compile Remove="ReorderParameters\ReorderParametersTests.cs" />
-    <Compile Remove="ReorderParameters\ReorderParametersTests.InvocationErrors.cs" />
-    <Compile Remove="ReorderParameters\ReorderParametersTests.InvocationLocation.cs" />
```

Comment:
@CyrusNajmabadi The base class for ReorderParametersTests is lost. Could it be that this feature was later ported as ChangeSignature feature?

There are similar test files in https://github.com/dotnet/roslyn/tree/2b571d1c7d1b1a32d8b1ac15299a8077818e2fab/src/EditorFeatures/CSharpTest/ChangeSignature. So I suspect this was left over from a previous port of the article. Will delete them as they are not useful currently.

Reply:
Are we sure there's more or less equivalent coverage here? It looks like the ItemGroup was added during the change to move us to file globbing, so I'm guessing the history here is:

1. The files weren't included in the project file, likely due to a mistake.
2. That mistake got immortalized into the Compile Removes, which perhaps kept things the same but made the problem more visible.

If I look at the ReorderParametersTests.InvocationErrors file, I see it testing for errors and I don't see those strings being tested for anywhere else, at least if I search for the resx name.

Reply:
@jasonmalinowski See https://github.com/dotnet/roslyn/blob/aa91123c53d9a84daf6053c69e88fff255c236d4/src/EditorFeatures/CSharpTest/ChangeSignature/ReorderParametersTests.InvocationErrors.cs for the equivalent of the deleted `ReorderParametersTests.InvocationErrors`

The difference is that the deleted one tests against the error message shown to the user from resx, while the existing one tests against the FailureKind enum

https://github.com/dotnet/roslyn/blob/e35ee860a5e236255e11ce1d72bd117432aeb5f3/src/EditorFeatures/Core/Implementation/ChangeSignature/AbstractChangeSignatureCommandHandler.cs#L150-L160

Reply:
Ah I got it, we have both:

src/EditorFeatures/CSharpTest/ChangeSignature/ReorderParametersTests.InvocationErrors.cs
src/EditorFeatures/CSharpTest/ReorderParameters/ReorderParametersTests.InvocationErrors.cs

I missed the bit of the directory name being different.